kubetest2-gke/deployer/build/gke_make.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2022 The Kubernetes Authors.
```

Comment:
this should remain the original year. I can't find anything written down but throughout the project we operate this way unless it is a generated file

Reply:
Done